lms/static/scripts/frontend_apps/services/vitalsource.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
I wonder if a module-level comment could be helpful to indicate that this entire module is fake/prototype-y at this point. Then again, the method comments cover this pretty well.

It's very nice overall, however, that this service module has been extracted out of the overcrowded API util module 👍🏻

Reply:
My brain kind of wants to see (regarding the naming of this module) `vital-source`, but, pedantically, _VitalSource_ _is_ one word, so I suppose `vitalsource` is correct here... (i.e. I think you got it right)

At some point I wonder if we'll end up with enough of a collection of "integrations" service modules to categorize them as such (subdirectory, naming convention) but that is, of course, premature at this point.